src/i18n/en/translation.en.json

```diff
@@ -8,7 +8,6 @@
     "alternativesubheader": "Need help? Find relevant users, organisations, groups and not least Challenges!",
     "body": "",
     "organization": {
-      "no-tags": "No tags available",
```

Comment:
this one is used in `OrganizationPopUp.tsx`

![image](https://user-images.githubusercontent.com/3524687/133139507-36a2e464-ca1d-42e3-b045-d6381860e1e7.png)